tools/Microsoft.Health.Tools.Sql.Tasks/Microsoft.Health.Tools.Sql.Tasks.csproj

```diff
@@ -1,14 +1,14 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net462</TargetFramework>
```

Comment:
Again, we don't want to use .NET Framework, right? Alternatively we could multi-target if we needed to, but I see this package has three targets: https://www.nuget.org/packages/Microsoft.SqlServer.DacFx/160.6296.0

Reply:
Also, what's the difference between this package and the old one?